service/src/test/resources/application-test.yml

```diff
@@ -48,3 +48,6 @@ cbas:
 pipelines:
   ingress:
     domainName: "some-tsps-domain.com"
+  sentry:
+    dsn: https://public@sentry.example.com/1
```

Comment:
do these get used anywhere?

Reply:
they get used when sentry is instantiated i.e. it has to look like this pattern otherwise it'll throw an exception